src/mysqlsh_helpers.py

```diff
@@ -140,11 +266,22 @@ def _wait_until_mysql_connection(self) -> None:
         Retry every 5 seconds for 30 seconds if there is an issue obtaining a connection.
         """
         commands = (
-            f"my_shell = shell.connect('{self.cluster_admin_user}:{self.cluster_admin_password}@{self.instance_address}')",
+            f"my_shell = shell.connect('root:{self.root_password}@{self.instance_address}')",
```

Comment:
it is simpler to check if socket file was created - such check don't need auth

Reply:
Can't the socket survive a daemon crash?

Reply:
confirmed that the socket file does not exist if the service is stopped. will be changing to check if socket exists in future commit

Reply:
changed to check if socket file exists in [37db8b5](https://github.com/canonical/mysql-operator/pull/6/commits/37db8b57689305a169c643d92613f15416ef8710)